activities/chris.json

```diff
@@ -0,0 +1,20 @@
+{
+  "tasks": [
+    {
+      "layers": [
+        ["d.rast", "map=slope"]
```

Comment:
You compute two curvatures, but then show slope (which happens to be available in the dataset).